tools/delete_imaging_upload.pl

```diff
@@ -335,6 +356,7 @@ =head2 Methods
     (@uploadID > 1 ? 'Uploads' : 'Upload'),
     $uploadIdsString
 );
```

Comment:
Here, if you used the `-type` option, it would be good to update the message to say that "the scan types X, Y, Z of the upload ID A were successfully deleted"